alma/alma-woocommerce-gateway
# Blocks display pay now

## Reason for change



Display correctly with design the pay now block.

[Linear task](https://linear.app/almapay/issue/ECOM-1404/[💅-wc-blocks]-in-page-pay-now-blocks-are-displayed)

### Code changes



### How to test

_As a reviewer, you are encouraged to test the PR locally._
On woocommerce 8-5-1 or 8-6-0, on the checkout page, you should be able to see the new block.
![image](https://github.com/alma/alma-woocommerce-gateway/assets/43644110/116e8d77-4dc1-40c3-a496-0337bab234b0)




### Checklist for authors and reviewers




### Non applicable
